manager/sinks.go

```diff
@@ -22,9 +22,9 @@ import (
 	sink_api "github.com/GoogleCloudPlatform/heapster/sinks/api/v1"
 )
 
-func newSinks() ([]sink_api.ExternalSink, error) {
+func newSinks(sinkUris []string) ([]sink_api.ExternalSink, error) {
```

Comment:
Agreed on the string stuff. I saw that `Uris` was just `[]string`, so I went with that. I've worked with the `flag.Value` interface before, I could get that working.

Haven't looked into sink configs yet, but I guess this is for exporting them on a `GET /api/v1/sinks`, right?

I'd actually prefer to do all of this in a separate PR if that's OK. Since this is just continuing with what we were doing already, really.